### ISSUE TITLE
The field names in java and python are different

### ISSUE THREAD

**Issue by maxhutch** [CitrineInformatics/api-documentation#2]
Some of the documentation is shared between java and python, but they use different naming conventions.  For example, the `PifSearchResult` has a `totalNumHits` member [in the docs](https://citrineinformatics.github.io/api-documentation/api/search/pif/result/PifSearchResult.html) but that is called `total_num_hits` in the python client.

**Comment by kjaym** [comment]
To be consistent with all of the other naming in the python package, the field names are in snake case. Should we just mention that snake case is used throughout the python client?

**Comment by maxhutch** [comment]
Could we do it on each page?  I assume users will be landing on the api docs via search, not just reading through them.

**Comment by kjaym** [comment]
Of course. Maybe put a star next to field names that have camel case in them and just note under the table what they correspond to in python.
